packages/terra-tabs/src/clinical-lowlight-theme/StatusLayout.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-status-layout-padding-bottom: 0.714rem;
```

Comment:
why the names are reflecting application status ?

Reply:
Prefixed with "--terra-tabs" in https://github.com/cerner/terra-framework/pull/1693/commits/72db1fd101ce8c28b1afa9da2f2acddce6df20f6